service/model/project_invitation.go

```diff
@@ -0,0 +1,119 @@
+package model
+
+import (
+	"errors"
+	"time"
+
+	cryptox "release-manager/pkg/crypto"
+	validator "release-manager/pkg/validator"
+
+	"github.com/google/uuid"
+)
+
+const (
+	ProjectRoleEditor ProjectRole = "editor"
+	ProjectRoleViewer ProjectRole = "viewer"
+)
```

Comment:
Only two roles are necessary at the project level because all "admin" actions are granted only to admin users.